app/bitbucket.yml

```diff
@@ -26,6 +26,7 @@ services:
       - python util/jtl_convertor/jtls-to-csv.py kpi.jtl selenium.jtl
     post-process:
       - python util/analytics.py bitbucket
+      - python util/cleanup_result_dir.py
```

Comment:
Let's rename the script to `cleanup_results_dir.py`

Reply:
Yes, you are right